### PR TITLE
[zh-cn]: sync translation for *Broadcast Channel API*

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

https://developer.mozilla.org/en-US/docs/Web/API/Broadcast_Channel_API

### Related issues and pull requests

part of the #21059

### CODE REVIEW

File: files/zh-cn/web/api/broadcast_channel_api/index.md [hunk 1]
```diff
@@ -1,65 +1,72 @@
...
 ### 接收消息
 
-当消息被发送之后，所有连接到该频道的 {{domxref("BroadcastChannel")}} 对象上都会触发 [`message`](/zh-CN/docs/Web/API/BroadcastChannel/message_event) 事件。该事件没有默认的行为，但是可以使用 {{domxref("BroadcastChannel.onmessage", "onmessage")}} 事件处理程序来定义一个函数来处理消息。
+发布消息时，会向连接到此频道的每个 {{domxref("BroadcastChannel")}} 对象发送一个 [`message`](/zh-CN/docs/Web/API/BroadcastChannel/message_event) 事件。可以使用 {{domxref("BroadcastChannel/message_event", "onmessage")}} 事件处理器为此事件运行一个函数：
 
 ```js
-// 简单示例，用于将事件打印到控制台
-bc.onmessage = function (ev) {
-  console.log(ev);
+// 仅将事件记录到控制台的处理程序：
```
Comment (yin1999) [inline files/zh-cn/web/api/broadcast_channel_api/index.md:45]:
```suggestion
// 仅将事件记录到控制台的处理器：
```

File: files/zh-cn/web/api/broadcast_channel_api/index.md [hunk 2]
```diff
@@ -1,65 +1,72 @@
...
 
-Broadcast Channel API 是一个非常简单的 API，内部包含了跨上下文通讯的接口。它可用于检测同源网站环境中其他浏览器选项卡下的用户操作，例如当用户登录到帐户时。没有定义消息传输协议，故不同上下文中的不同文档需要自己实现它：规范没有对此提出协议或要求。
+Broadcast Channel API 的独立接口允许跨上下文通信。它可用于检测同一来源内其他标签页中的用户操作，例如用户登录或退出时。
+
+消息传递协议未定义，不同的浏览上下文需要自行实现；规范中没有协商或要求。
+
+## 接口
+
+- {{domxref("BroadcastChannel")}}
+  - : 表示任何给定{{glossary("origin", "源")}}的{{glossary("browsing context", "浏览上下文")}}都可以订阅的命名频道。
```
Comment (yin1999) [inline files/zh-cn/web/api/broadcast_channel_api/index.md:69]:
```suggestion
  - : 表示任何给定{{glossary("origin", "源")}}的{{glossary("browsing context", "浏览上下文")}}都可以订阅的具名频道。
```

**Review by yin1999 — COMMENTED** [review]
nits:

File: files/zh-cn/web/api/broadcast_channel_api/index.md [hunk 3]
```diff
@@ -1,65 +1,72 @@
...
 ---
 
-{{DefaultAPISidebar("Broadcast Channel API")}}
+{{DefaultAPISidebar("Broadcast Channel API")}} {{AvailableInWorkers}}
 
-**Broadcast Channel API** 可以实现同 {{glossary("origin", "源")}} 下浏览器不同窗口，Tab 页，frame 或者 iframe 下的 {{glossary("browsing context", "浏览器上下文")}} (通常是同一个网站下不同的页面) 之间的简单通讯。
+**Broadcast Channel API** 允许{{glossary("browsing context", "浏览上下文")}}（即 _window_、_tab_、_frame_ 或 _iframe_）与同{{glossary("origin", "源")}}的 worker 之间进行基本通信。
 
-{{AvailableInWorkers}}
+通过创建 {{domxref("BroadcastChannel")}} 对象，你可以接收发布到该对象的任何消息。你不必维护对要与之通信的框架或工作者的引用：他们可以通过构建具有相同名称的 {{domxref("BroadcastChannel")}} 来“订阅”特定频道，并在它们之间进行双向通信。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/broadcast_channel_api/index.md:12]:
```suggestion
通过创建 {{domxref("BroadcastChannel")}} 对象，你可以接收发布到该对象的任何消息。你不必维护对要与之通信的框架或 worker 的引用：他们可以通过构建具有相同名称的 {{domxref("BroadcastChannel")}} 来“订阅”特定频道，并在它们之间进行双向通信。
```

File: files/zh-cn/web/api/broadcast_channel_api/index.md [hunk 4]
```diff
@@ -1,65 +1,72 @@
...
+{{DefaultAPISidebar("Broadcast Channel API")}} {{AvailableInWorkers}}
 
-**Broadcast Channel API** 可以实现同 {{glossary("origin", "源")}} 下浏览器不同窗口，Tab 页，frame 或者 iframe 下的 {{glossary("browsing context", "浏览器上下文")}} (通常是同一个网站下不同的页面) 之间的简单通讯。
+**Broadcast Channel API** 允许{{glossary("browsing context", "浏览上下文")}}（即 _window_、_tab_、_frame_ 或 _iframe_）与同{{glossary("origin", "源")}}的 worker 之间进行基本通信。
 
-{{AvailableInWorkers}}
+通过创建 {{domxref("BroadcastChannel")}} 对象，你可以接收发布到该对象的任何消息。你不必维护对要与之通信的框架或 worker 的引用：他们可以通过构建具有相同名称的 {{domxref("BroadcastChannel")}} 来“订阅”特定频道，并在它们之间进行双向通信。
 
-广播频道会被命名和绑定到指定的源。
+![Broadcast Channel API 的原理](broadcastchannel.png)
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/broadcast_channel_api/index.md:14]:
```suggestion
![广播频道 API 的原理](broadcastchannel.png)
```

**Review by yin1999 — APPROVED** [review]
Thank you @skyclouds2001

**Pull request merged**